src/services/template_converter_service.py

```
@@ -0,0 +1,105 @@
+        self.temp_dir.mkdir(exist_ok=True)
+
+    def convert_template_to_pdf(self, template: SlideTemplate) -> bytes:
+        markdown_content = template.read_markdown_content()
+        temp_md_path = self.temp_dir / f"{template.id}.md"
+        temp_pdf_path = self.temp_dir / f"{template.id}.pdf"
+
+        try:
+            temp_md_path.write_text(markdown_content, encoding="utf-8")
+
+            command = ["marp", str(temp_md_path), "-o", str(temp_pdf_path)]
+            result = subprocess.run(
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)